### PR TITLE
[PWGMM] Updated dndeta-mft using fwdtrack-to-collision-associator 

### PR DESCRIPTION
**Pull request by sarahherrmann**
Also fix in fwdtracktoCollisionAssociator to have possibly doprocessFwdAssocWithTime==doprocessFwdStandardAssoc=false

### CODE REVIEW

File: Common/TableProducer/fwdtrackToCollisionAssociator.cxx [hunk 1]
```diff
@@ -45,8 +45,8 @@ struct FwdTrackToCollisionAssociation {
 
   void init(InitContext const&)
   {
-    if (doprocessFwdAssocWithTime == doprocessFwdStandardAssoc || doprocessMFTAssocWithTime == doprocessMFTStandardAssoc) {
-      LOGP(fatal, "Exactly one process function between standard and time-based association should be enabled!");
+    if ((int(doprocessFwdAssocWithTime) + int(doprocessFwdStandardAssoc) > 1) || (int(doprocessMFTAssocWithTime) + int(doprocessMFTStandardAssoc) > 1)) {
```
Comment (aalkin) [inline Common/TableProducer/fwdtrackToCollisionAssociator.cxx:48]: What is the purpose of this change?

Comment (sarahherrmann) [reply Common/TableProducer/fwdtrackToCollisionAssociator.cxx:48]: I want to have the possibility to have oprocessFwdAssocWithTime == doprocessFwdStandardAssoc = false

Comment (sarahherrmann) [reply Common/TableProducer/fwdtrackToCollisionAssociator.cxx:48]: Exactly :)

Comment (aalkin) [reply Common/TableProducer/fwdtrackToCollisionAssociator.cxx:48]: I see, then I recommend to separate the conditions and do `fatal` only when both fwd processes are enabled.

Comment (sarahherrmann) [reply Common/TableProducer/fwdtrackToCollisionAssociator.cxx:48]: Ok !

Comment (aalkin) [reply Common/TableProducer/fwdtrackToCollisionAssociator.cxx]: You also need to make sure that at least on process function is enabled by these flags, or add a "dummy" `process` that does nothing.

**Review by aalkin — APPROVED** [review]

**Comment by sarahherrmann** [comment]
Hi all, required tests are all successful, do you think it could be merged soon ? I would like to run it on hyperloop tomorrow or the day after tomorrow if possible
Thanks in advance and sorry for the rush

File: PWGMM/Mult/Tasks/dndeta-mft.cxx [hunk 2]
```diff
@@ -83,37 +89,31 @@ struct PseudorapidityDensityMFT {
...
-      {"Tracks/Control/ReassignedVertexCorr", "; #it{z}_{vtx}^{orig} (cm); #it{z}_{vtx}^{re} (cm)", {HistType::kTH2F, {ZAxis, ZAxis}}} //
-    }                                                                                                                                  //
+    } //
   };
 
   void init(InitContext&)
   {
+
+    if (int(doprocessMult) + int(doprocessMultReassoc) + int(doprocessCountingCentrality) > 1) {
+      printf(">>>>>> doprocessMult = %d, doprocessMultReassoc =%d, doprocessCountingCentrality=%d\n", int(doprocessMult), int(doprocessMultReassoc), int(doprocessCountingCentrality));
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft.cxx:113]: Please do not use `printf` and use `static_cast<int>()`.

**Review by aalkin — CHANGES_REQUESTED** [review]

File: PWGMM/Mult/Tasks/dndeta-mft.cxx (hunk 2 above)
Comment (sarahherrmann) [reply PWGMM/Mult/Tasks/dndeta-mft.cxx:113]: Ok !

**Review by aalkin — APPROVED** [review]

**Review by jgrosseo — APPROVED** [review]
